proxy/src/http.rs

```diff
@@ -101,6 +103,7 @@ where
 
     let api = path("v1").and(combine!(
         avatar_filter,
+        account_filter,
```

Comment:
For reviewers: I added this here instead of before `avatar_filter` because doing the latter failed to compile. Since we will merge `avatar_filter` into `ids`, I just put it here for now.

Reply:
Addressed in 8953b2f2a1466b2871e2810d128ff7926c73e88a